apps/api/src/modules/summary/entities/summary.entity.ts

```diff
@@ -0,0 +1,52 @@
+import { ObjectType, Field } from '@nestjs/graphql';
+import { Entity, Column, PrimaryGeneratedColumn, UpdateDateColumn, CreateDateColumn } from 'typeorm';
+import { IsEnum, IsOptional } from 'class-validator';
+import { JobStatus, Status } from 'src/common/constants/summary';
+
+@Entity({ name: 'jobs' })
+@ObjectType()
+export class Summary {
+  @PrimaryGeneratedColumn()
+  @Field()
+  id?: number;
```

Comment:
Why have you made everything as Optional?
This is an entity file. It will contain details of the Entity.

Reply:
This was done for testing the fix for non-nullable field error while running query in graphQL playground. Will remove the optional part from entity.